frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -0,0 +1,133 @@
+Feature: Monitoring tab on the Sidebar and Health Checks
+    As a user, I should be able to see Monitoring tab on the sidebar and add Health Checks
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke
+Scenario: Monitoring tab on the Sidebar for Deployments
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    Then user will see View Monitoring dashborad link
+    And user will see CPU Usage Metrics
+    And user will see Memory Usage Metrics
+    And user will see Receive Bandwidth Metrics
+    And user will see All Events dropdown
+
+
+@smoke
+Scenario: Monitoring tab on the Sidebar for Deployments
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    And user clicks on View Monitoring dashborad link
+    Then user will be taken to Dashboard tab on the Monitoring page
+
+
+@smoke
+Scenario: Health Checks option in Advanced Options: MH-04-TC01
+    Given user is at +Add page
+    When user clicks on From Git card
+    Then user will see Health Checks advanced option
+
+
+@smoke
+Scenario: Health Checks option in Advanced Options: MH-04-TC02
```

Comment:
Add test data for the scenario

Reply:
We haven't updated anything in the Probe. That's why there is no need of the test data.

Reply:
You are using Git url and name of the application in the use case so you need to add test data for that